style.css

```
@@ -0,0 +1,82 @@
+@import url('https://fonts.googleapis.com/css?family=Bitter');
+
+html,
+body {
+  font-size: 100%; }
+
+body {
+  background: #f7f7f7;
+  color: #222222;
+  padding: 0;
+  margin: 0;
+  font-family: 'Bitter', serif;
+  font-weight: normal;
+  font-style: normal;
+  line-height: 1;
+  position: relative;
+  text-align: center;
+}
+
+.h1 {
+    color: #e87c36 !important;
```

i've tried pretty hard to make orange mean clickable in _all_ cases on nkd.su, so i have to admit i am not a fan of this or the `.h2`

---

Beat me by seconds lol.

---

```suggestion
    color: white !important;
```
I completely forgot this was the design intention. A change to white should be suitable enough?